<launch>
  <!-- px4_ctl node-->
     <node pkg="px4_ctl" type="mavros_topic" name="mavros_topic" output="screen" >
     <!--param name=" " value=" " type=" "-->
     <param value="0.0" name="px4start_x" type="double"/>
     <param value="0.0" name="px4start_y" type="double"/>
     <param value="3.0" name="px4start_z" type="double"/>
     
     <param value="0.0" name="px4start_orix" type="double"/>
     <param value="0.0" name="px4start_oriy" type="double"/>
     <param value="0.0" name="px4start_oriz" type="double"/>
     <param value="1.0" name="px4start_oriw" type="double"/>
     
     </node>



</launch>